packages/react/react-types.d.ts

```diff
@@ -0,0 +1,17 @@
+import type { Size } from '@digdir/designsystemet-react';
```

Comment:
We dont export this type from the library so this import causes a type error.

Reply:
Good point! Fixed.